src/Components/movie-view/movie-view.jsx

```diff
@@ -1,11 +1,17 @@
-import PropTypes from "prop-types";
-import Button from "react-bootstrap/Button";
+import { useParams } from "react-router-dom";
+import { Link } from "react-router-dom";
+import "./movie-view.scss";
```

Comment:
You added this line, without adding the file. Remove it if you don't yet have the file